articles/sentinel/tutorial-monitor-your-data.md

```diff
@@ -62,9 +62,9 @@ This tutorial helps you visualize your data in Azure Sentinel.
 
 1. To edit the workbook, select **Edit**, and then add text, queries, and parameters as necessary. For more information on how to customize the workbook, see how to [Create interactive reports with Azure Monitor Workbooks](../azure-monitor/platform/workbooks-overview.md). 
 
-1. When building a query, set the **Data source** is set to **Logs**, the **Resource type** is set to **Log Analytics** and then choose the relevant workspace(s). 
+1. When building a query, make sure the **Data source** is set to **Logs**, the **Resource type** is set to **Log Analytics** and then choose the relevant workspace(s). 
```

Comment:
after **Logs**, there should be the word "and" instead of a comma, and there should be a comma after **Log Analytics**".

Reply:
Thanks, fixed.